src/components/common/Img/Img.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import placeholder from '../../../assets/images/placeholder.png';
+
+const Img = ({ imgId, imgClass, imgSrc, alt, width, height }) => (
+  <img id={imgId} src={imgSrc} alt={alt} className={imgClass} style={{ width, height }} />
+);
+
+Img.propTypes = {
+  imgId: PropTypes.string,
+  imgSrc: PropTypes.string,
+  imgClass: PropTypes.string,
+  alt: PropTypes.string,
+  width: PropTypes.string,
+  height: PropTypes.string
+};
+
+Img.defaultProps = {
+  imgClass: 'image',
```

Comment:
@luc-tuyishime it's better to have this format for images ` <div className="image"> <img .../></div>`

`.image` is used to wrap the `img` tag.

Reply:
I don't think so, the better way is to call the Img component and use it by passing in properties instead of creating many divs.

Reply:
However, still, it's not good! You can check how images are styled in styles.scss.

Reply:
So why do we have an Img component now?